Add collection proposal template. (#1362)
GoogleChrome/web.dev: .github/ISSUE_TEMPLATE/collection_proposal.md

---

> name: Propose a new collection

Reviewer: Should we change other template to "Propose a new post/codelab" for clarity?

> about: Use this ticket when you are submitting new collection for review.

Reviewer: ```suggestion
about: Use this ticket when you are submitting new a collection for the site. If this doesn’t look right, [choose a different type](https://github.com/GoogleChrome/web.dev/issues/new/choose).
```
Should the other ticket read "to propose a new post/codelab" instead of "content" for clarity?

Author: I think github automatically inserts that "If this doesn't look right..." text for us. I looked and none of the other templates contain that text.

title: 'collection: <Add your title>'
labels: collection proposal
assignees: ''

---

> **Reviewer**: [Leave blank. web.dev team will assign a reviewer.]

Reviewer: Is there a reason you omitted the author field?

Author: added it back

**Process**

> When you're ready, drop a link to your proposal doc in the 1st step below and

Reviewer: ```suggestion
Drop a link to your proposal doc in the 1st step below and
```

check the box. A member of the web.dev team will assign this ticket to

> themselves and work as your reviewer to help you complete the remaining steps.

Reviewer: ```suggestion
a reviewer, who will help you complete the remaining steps.
```
To fix the number agreement error. Needs fixed in content proposal template also.
